applications/crossbar/priv/couchdb/schemas/phone_numbers.json

```diff
@@ -84,6 +84,8 @@
                 },
                 "region": {
                     "description": "The region (state) where the number is in service",
+                    "maxLength": 2,
+                    "minLength": 2,
```

Comment:
yeah so about this: am I right in assuming "every" e911 provider uses the 2-letter format for states?
I couldn't find info on Dash's handling of Canada regions (and we need this)